rpc/rpc.go

```diff
@@ -20,7 +20,7 @@ import (
 // if the call is continued, meaning the underlying channel will be kept open for either
 // streaming back more results or using the channel as a full duplex byte stream.
 type Caller interface {
-	Call(ctx context.Context, selector string, params, reply interface{}) (*Response, error)
+	Call(ctx context.Context, selector string, params any, reply ...any) (*Response, error)
```

Comment:
@progrium how do you feel about changing these interfaces? Most calling code should continue to work (e.g. the tests and things didn't need to change their `Call()` invocations.
However, it does affect some things like people implementing this or the `Responder` interface. We could add new interfaces for the new signatures. Trying to infer it from a `[]any` value seems like it would come with some ambiguity, or at least be a little less explicit, so I would rather avoid that.
I'm not sure how widely used this is to avoid potential breaking changes, but I guess it would be easier to do sooner than later.

Reply:
We are the only users afaik so we can change anything that has a problem with it. I think it's a good idea.